web_interface/app.py

```
@@ -590,7 +590,8 @@ def logs_generator():
             # Note: User should be in systemd-journal group to read logs without sudo
             try:
                 result = subprocess.run(
-                    ['journalctl', '-u', 'ledmatrix.service', '-n', '50', '--no-pager'],
+                    ['journalctl', '-u', 'ledmatrix.service', '-u', 'ledmatrix-web.service',
+                     '-n', '50', '--no-pager', '--output=short-iso'],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)